### PR TITLE
Release/v1

### PR DESCRIPTION
**Pull request by johngwilkinson**
Tested with assigned-learning-service (using @release/v1 to hit this branch). Will tag as v1 after merge.

### CODE REVIEW

**Review by SheranWiratungaD2L — APPROVED** [review]

File: ci-setup/action.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+name: ci-setup
+description: Set up node, terraform, and aws profile
+
+inputs:
+  role-arn:
+    description: 'AWS IAM role to assume'
```
Comment (AntonBazhal) [inline ci-setup/action.yml:6]:
```suggestion
    description: AWS IAM role to assume
```

File: ci-setup/action.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  node-version:
+    description: 'Node version to install'
+    required: false
+    default: 12.x
+  terraform-version:
+    description: 'Terraform version to install'
+    required: false
+    default: 1.0.0
+  npm-registry-url:
+    description: npm registry to use for d2l packages
```
Comment (AntonBazhal) [inline ci-setup/action.yml:20]:
```suggestion
    description: NPM registry to use for D2L packages
```

File: ci-setup/action.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    required: true
+  node-auth-token:
+    description: read-only token for the above npm registry
+    required: true
+  aws-access-key-id:
+    description: aws access key id
```
Comment (AntonBazhal) [inline ci-setup/action.yml:26]:
```suggestion
    description: AWS access key id
```

**Review by AntonBazhal — APPROVED** [review]
Minor wording

**Pull request merged**